os/arch/arm/src/armv7-a/arm_prefetchabort.c

```diff
@@ -80,7 +84,7 @@ static inline void print_prefetchabort_detail(uint32_t *regs, uint32_t ifar, uin
 	lldbg_noarg("\n");
 	_alert("#########################################################################\n");
 	_alert("PANIC!!! Prefetch Abort at instruction : 0x%08x\n",  regs[REG_PC]);
-	_alert("PC: %08x IFAR: %08x IFSR: %08x\n", regs[REG_PC], ifar, ifsr);
```

Comment:
I beleive we print all register values also after stackdump, so is this required?

```
===========================================================
Asserted task's register dump
===========================================================
arm_registerdump: R0: 0e676eb0 00000001 00000030 00000000 63f853a0 0e59e76d 60fd2e60 6101c230
arm_registerdump: R8: 72a9f766 00000194 0e5fa9d9 00000000 00000000 61109328 0e163c0d 0e552fea
arm_registerdump: CPSR: 20000070
```

Reply:
I think yes.
If we don't need to add lr here, I think we don't need to print pc either with the same reason.

But it's the title to show where the problem causes.
It can be pr or lr, then we should print both.

Except the system engineer, noone checks the content below...